capstone/src/main/java/com/google/step/clients/MetricsClient.java

```diff
@@ -128,9 +128,11 @@ public WeeklyPageView getCurrentPageViews(String restaurantKey) {
         // Map to maintain a reference to each restaurantList and make searching for them easy.
         // The map and the list share the reference to each restaurantPageViews.
         Map<String, List<WeeklyPageView>> restaurantNameMap = new HashMap<>();
+        Map<String, String> restaurantIdMap = new HashMap<>();
```

Comment:
Nit: Consider making a private inner class that stores a restaurant name and a List<WeeklyPageView>.
Then you can have a single map from the restaurant id (which is guaranteed to be unique for each restaurant) to this class.

Reply:
Done! It looks a lot better and less confusing! Thanks for the tip!